presto-spi/src/main/java/com/facebook/presto/spi/StandardErrorCode.java

```diff
@@ -118,6 +118,7 @@ public enum StandardErrorCode
     UNSUPPORTED_ANALYZER_TYPE(0x0001_0029, INTERNAL_ERROR),
     SLICE_TOO_LARGE(0x0001_002A, INTERNAL_ERROR),
     HBO_FAILURE(0x0001_002B, INTERNAL_ERROR),
+    CLUSTER_TASK_COUNT_TOO_HIGH(0x0001_002C, INTERNAL_ERROR),
```

Comment:
may be call it `CLUSTER_HAS_TOO_MANY_RUNNING_TASK` to align it with other error code and also more self explanatory.